beacon-chain/blockchain/block_processing.go

```diff
@@ -162,13 +162,16 @@ func (c *ChainService) runStateTransition(
 		block,
 		headRoot,
 		&state.TransitionConfig{
-			VerifySignatures: false, // We disable signature verification for now.
-			Logging:          true,  // We enable logging in this state transition call.
+			VerifySignatures:      false, // We disable signature verification for now.
+			Logging:               true,  // We enable logging in this state transition call.
+			InvalidDepositRemoval: true,  // We enable this to remove invalid deposits.
```

Comment:
I dont think we need this flag, we should **always** be removing invalid deposit anyway.

Thoughts? @rauljordan @prestonvanloon

Reply:
we actually do need this flag, because if we dont have it and run this on another state transition which is not part of the main chain service ex: stategen,rpc service, it might end up modifying the 
`depositsToRemove` variable which we check to know which deposits to remove from the db.